managed/src/main/java/com/yugabyte/yw/controllers/AccessKeyController.java

```diff
@@ -139,22 +122,17 @@ public Result create(UUID customerUUID, UUID providerUUID) {
       }
     } catch(RuntimeException | IOException e) {
```

Comment:
Find the source of runtime exception and proper treatment like mentioned in the other review.